### PR TITLE
Fix #100536 [Feature Request] pitch up/down feature in the piano roll.

### PR DESCRIPTION
**Pull request by luyikei**
https://musescore.org/en/node/100536#comment-447531

### CODE REVIEW

**Comment by luyikei** [comment]
I also implemented an up/down octave feature.

**Comment by lasconic** [comment]
could you squash the two commits? https://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

It would be cool if we could reuse the MuseScore shortcuts/actions instead if creating new ones with new shorcuts.

**Comment by luyikei** [comment]
before_delete
after_delete

I don't know how we should handle rests after we delete notes. In this patch, I make rests for them, as the default behavior does.

**Comment by luyikei** [comment]
I think we should implement a range selection for the piano roll

File: mscore/pianoroll.cpp [hunk 1]
```diff
@@ -206,10 +206,31 @@ PianorollEditor::PianorollEditor(QWidget* parent)
             }
 
       QActionGroup* ag = new QActionGroup(this);
-      QAction* a = new QAction(this);
-      a->setData("delete");
-      a->setShortcut(Qt::Key_Delete);
-      ag->addAction(a);
+      QAction* deleteAction = new QAction(this);
```
Comment (lasconic) [inline mscore/pianoroll.cpp]: What about replacing this by
```
ag->addAction(getAction("pitch-up"));
ag->addAction(getAction("pitch-down"));
ag->addAction(getAction("pitch-up-octave"));
ag->addAction(getAction("pitch-down-octave"));
```

**Comment by luyikei** [comment]
Since we can only select notes here, implementing rage selection will be difficult I think

**Comment by luyikei** [comment]
The deletion of notes problem is actually not so related to this patch, pitch up/down feature, so I'll create another issue on the MuseScore issue tracker. And I'll revert the code around deletion as it used to be.

**Comment by lasconic** [comment]
Looks like we broke something with this one... https://musescore.org/en/node/104881

**Pull request merged**